src/connection_wrap.cc

```diff
@@ -99,28 +92,22 @@ void ConnectionWrap<WrapType, UVType>::AfterConnect(uv_connect_t* req,
     writable = uv_is_writable(req->handle) != 0;
   }
 
-  Local<Value> argv[5] = {
-    Integer::New(env->isolate(), status),
-    wrap->object(),
-    req_wrap->object(),
-    Boolean::New(env->isolate(), readable),
-    Boolean::New(env->isolate(), writable)
-  };
+  Local<Value> argv[5] = {Integer::New(env->isolate(), status),
```

Comment:
ouch

Reply:
@devsnek Yeah I thought it looked strange but then this came from Google's style and in the documentation they argued this is more suited for C++11 (This is named `Cpp11BracedListStyle`)